src/python/bcc/__init__.py

```diff
@@ -1093,7 +1093,12 @@ def cleanup(self):
             # non-string keys here include the perf_events reader
             if isinstance(k, str):
```

Comment:
I see, that makes sense. I was wondering before why would people use the polling to get callchain:)
Maybe we should have `kprobe_poll` vs `perf_buffer_poll` then?